devcontainer/postCreate.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -euo pipefail
+
+/usr/local/bin/pip3 install --user -r .devcontainer/requirements.in
```

Comment:
Shouldn't this be `/opt/devcontainer` not `.devcontainer`?

Reply:
Yep, didn't see this before merging! I did have it, then reorganised commits and lost the change.

Fixed in #59.